app/javascript/lib/study-overview/explore-single.js

```diff
@@ -73,7 +83,12 @@ function attachEventHandlers(study, gene) {
 export default async function exploreSingle() {
   // As set in exploreDefault
   const study = window.SCP.study
-  const gene = window.SCP.gene
+  let gene = window.SCP.gene
+  if (!gene || !gene.length) {
+    // if window.SCP.gene is not defined, read it from the search bar
+    // this happens when, e.g. 'view as' -> 'violin - mean' is selected
```

Comment:
This won't be sufficient context in a few months.  Easing maintenance:

```suggestion
    // this happens when, e.g. 'Collapse genes by' -> 'Mean' is selected
    // in "Advanced Gene Search" after clicking the hamburger menu to 
    // left of search box
```